dsgrid/registry/dataset_registry_manager.py

```diff
@@ -244,56 +299,50 @@ def _register(
             pivoted_columns = None
             pivoted_dimension_type = None
 
-        config.model.dataset_version = "1.0.0"
-        dataset_registry_dir = self.get_registry_data_directory(dataset_id)
-        if not dataset_registry_dir.parent.exists():
-            msg = (
-                f"The registry data path: {dataset_registry_dir.parent} does not exist "
-                "(at least from the current computer). Please contact the dsgrid team if this is "
-                "unexpected."
-            )
-            raise Exception(msg)
-
-        dataset_path = dataset_registry_dir / config.model.dataset_version
-        dataset_path.mkdir(exist_ok=True, parents=True)
-        self.fs_interface.mkdir(dataset_registry_dir)
         found_files = False
         for filename in ALLOWED_DATA_FILES:
             assert config.dataset_path is not None
             path = Path(config.dataset_path) / filename
             if path.exists():
-                dst = dataset_path / filename
+                name = os.path.splitext(filename)[0]
```

Comment:
NIT, I hear that you generally wanna stay in either the Path world or the os.path world. There's probably a Path method you can use for this instead.